app/models/broker.rb

```diff
@@ -20,4 +20,8 @@ class Broker < ApplicationRecord
 
   has_many :uploads, as: :user
 
+  def bid_limit_exceeded?
+    return account_type == 0 && bids.this_month.count >= 10
```

Comment:
We can still refactor :)

Let's extract `10` into a constant variable:

```
BID_LIMIT = 10
```

then replace it with the constant:

```
return account_type == 0 && bids.this_month.count >= BID_LIMIT
```

Then in our `bids_controller.rb`, we can do this:

```
flash[:alert] = "Sorry! You can't bid on more then #{current_user.class::BID_LIMIT} jobs please switch to premium plan."
redirect_to root_path
```

This way we can keep our code DRY :)

Same thing for `forward_freight.rb` ;)